src/test/java/org/apache/commons/collections4/bloomfilter/LayeredBloomFilterTest.java

```diff
@@ -311,4 +313,36 @@ public final void testNext() {
         assertFalse(filter.get(1).contains(TestingHashers.FROM11));
         assertTrue(filter.get(1).contains(new IncrementingHasher(11, 2)));
     }
+
+    @Test
+    public void testClean() {
```

Comment:
I think this test is made redundant by testing the method on the LayeredBloomFilter. In the interest of removing code duplication I would drop this and just leave the test calling the public API on the filter.

Reply:
Yes, drop the LayerManagerTest (as a duplicate). Sorry for the mistake.